File: mw/get_sigma_lbm_dm.m
function [sigma_lbm_dm] = get_sigma_lbm_dm(distmat)

% Compute the sigma_lbm with origin given by first location
sigma_lbm = get_sigma_lbm(distmat);

% Demean
sigma_lbm_dm = demean_sigma(sigma_lbm);

end

File: R/get_sigma_lbm_dm.R
#' Compute Demeaned Sigma LBm
#'
#' Computes the sigma_lbm matrix using the provided distance matrix and then demeans it.
#'
#' @param distmat A numeric matrix of distances.
#' @return A numeric matrix representing the demeaned sigma_lbm.
get_sigma_lbm_dm <- function(distmat) {
  sigma_lbm <- get_sigma_lbm(distmat)
  sigma_lbm_dm <- demean_sigma(sigma_lbm)
  return(sigma_lbm_dm)
}
